demisto_sdk/commands/lint/linter.py

```diff
@@ -328,6 +328,9 @@ def _gather_facts(self, modules: dict) -> bool:
             )
             self._facts["is_long_running"] = script_obj.get("longRunning")
             self._facts["commands"] = self._get_commands_list(script_obj)
+            self._facts["formatting_script"] = "indicator-format" in script_obj.get(
```

Comment:
the `formatting_script` value should come from a constant, as it might be relevant in other places (and might be expanded in the future).
Please extract it to the global variable, or replace the implementation with `is_formatting_script()` function.